low-level/framework/base/sspl_constants.py

```diff
@@ -36,6 +36,7 @@
 RESOURCE_PATH = "/opt/seagate/%s/sspl/low-level/json_msgs/schemas/" % (PRODUCT_FAMILY)
 CLI_RESOURCE_PATH = "/opt/seagate/%s/sspl/low-level/tests/manual" % (PRODUCT_FAMILY)
 DATA_PATH = "/var/%s/sspl/data/" % (PRODUCT_FAMILY)
+IEM_DATA_PATH = "/var/%s/sspl/data/iem/Loggged_Iem"  %(PRODUCT_FAMILY)
```

Comment:
directory name can be better, use small case

Reply:
Changed dir name to 'sspl_iem'.